### ISSUE TITLE
Compatibility Meteor 3.0.1

### ISSUE THREAD

**Issue by martijnbar** [TAPevents/tap-i18n#240]
It seems there are compatibility issues with Meteor version 3.0.1. I get the following error when trying to add the tap-i18n package (v.2.0.1) to my meteor 3.0.1 application

error: Conflict: Constraint webapp@1.10.1 is not satisfied by webapp 2.0.0.
Constraints on package "webapp":

* webapp@1.10.1 <- tap:i18n 2.0.1

**Comment by ghost** [comment]
Hi,
I have the same problem. Any idea to fix it?

**Comment by Marcel-Tronco** [comment]
Hi,
same same here. I just tried running the package with pinning the webapp version to 2.0.0 locally. It worked for me (in a setup with Blaze), but I can't verify that it works with other setups as there's no tests integrated in the repo.

**Comment by theosp** [comment]
The pinning was on the package.js level? If so, I'd appreciate a PR.

**Comment by Marcel-Tronco** [comment]
No, actually I pinned it in the `.versions`-file. The docs don't say to much about versioning in packages. They don't mention the `.version`-file anymore, so maybe it's just a compatibility feature for old packages. Anyways I just tried to delete the file altogether, because as you implied, nowadays versions in a package probably should be pinned in the package.js: and it still works.

**Comment by Marcel-Tronco** [comment]
The [changelog](https://v2-docs.meteor.com/changelog#v10220141219) says:
> The constraint solver no longer leaves a versions.json file in your packages source directories; when publishing a package that is not inside an app, it will leave a .versions file (with the same format as .meteor/versions) which you should check into source control.

That's explaining why local development made it seem superfluous anyways.
